docker-compose.yml

```diff
@@ -26,6 +26,7 @@ services:
        - MINIO_DOMAIN=minio
 
   app:
+    image: sync-engine_app
```

Comment:
Docker-compose bundled in newer image by default names images `sync-engine-app` (underscore versus hyphen). We want the old behavior so everything else keeps working.